src/qibolab/platform.py

```diff
@@ -285,6 +298,18 @@ def create_qubit_readout_pulse(self, qubit, start):
         qubit = self.get_qubit(qubit)
         return self.create_MZ_pulse(qubit, start)
 
+    def create_coupler_pulse(self, coupler, start, duration, amplitude):
+        pulse = CouplerFluxPulse(
+            start,  # + self.relative_start,
+            duration,
+            amplitude,
+            "Rectangular()",
+            channel=coupler.flux.name,
+            qubit=coupler.name,
+        )
+
+        return pulse
```

Comment:
If this is supposed to be calibrated we should consider adding it to the runcard and in native.py so that we can access it similarly to the other native pulses:
```py
self.couplers[coupler].native_gates...
```
Otherwise how will you create it using the calibrated `amplitude`/`duration`?

Reply:
I was assuming that it will always be part of a 2q gate so it got created there, but yeah I suppose it can be used in other situations as well.